samples/emoji-search/ios-uikit/EmojiSearchApp/EmojiSearchViewController.swift

```diff
@@ -37,7 +37,10 @@ class EmojiSearchViewController : UIViewController {
         let treehouseApp = emojiSearchLauncher.createTreehouseApp()
         let widgetSystem = EmojiSearchWidgetSystem()
         let treehouseView = Redwood_treehouse_hostTreehouseUIKitView(widgetSystem: widgetSystem)
-        let content = treehouseApp.createContent(source: EmojiSearchContent())
+        let content = treehouseApp.createContent(
+            source: EmojiSearchContent(),
+            codeListener: Redwood_treehouse_hostCodeListener()
```

Comment:
No default parameters calling into Kotlin from Swift is too bad. I’m _slightly_ tempted to do a manual `@JvmOverloads` style thing.